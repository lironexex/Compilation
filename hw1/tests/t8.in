// strings 1
"this is ok"
"this is a legal \t string"
"also legal \n\r\\"
"we can \"quote\" inside!"

