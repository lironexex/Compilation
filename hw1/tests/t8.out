1 COMMENT 1
2 STRING this is ok
3 STRING this is a legal 	 string
4 STRING also legal 
\
5 STRING we can "quote" inside!
